aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -58,6 +58,12 @@
 #define int_vector __m256i
 #endif // CPU_CAPABILITY_AVX512
 
+#if defined(__clang__)
+#  pragma clang diagnostic push
+#  pragma clang diagnostic ignored "-Werror"
```

Comment:
Please use `C10_CLANG_DIAGNOSTIC_PUSH` and `C10_CLANG_DIAGNOSTIC_IGNORE`, but agree with @albanD that ignoring warnings require some explanation